P5/Source/Guidelines/en/FT-TablesFormulaeGraphics.xml

```diff
@@ -559,14 +561,14 @@ discussion at .</note-->
       described in writings of various kinds. This applies to both historical
       and contemporary documents, even though methods of notating music have
       changed considerably in western history. In most cases, music notation
-      enters the text flow in a way similar to figures, images or graphs. On
+      enters the text flow in a way similar to figures, images, or graphs. On
       other occasions, elements of music notation are treated as inline
       characters in running text.</p>
     <p><gi>notatedMusic</gi> provides a way to signal the presence of music
       notation in text, but defer to other representations, which are not
       covered by the TEI guidelines, to describe the music notation itself. In
-      fact, several commercial, academic and standard bodies have developed
-      digital representations of music notation, and given the topic's
+      fact several commercial, academic, and standard bodies have developed
+      digital representations of music notation; and given the topic's
```

Comment:
@sydb I dissent with this use of the semicolon, and I think you should put the comma back. When you use "and" to link two parts of a compound sentence, the conjunction is preceded by a comma. You don't need a semicolon to disambiguate this "and" from the one at the end of the list earlier. If you were to change anything here, it might just be to split this into two sentences.

Reply:
Split sentence in two (in commit c22b97f6d, I think).